crm_phonecall_planner/wizards/crm_phonecall_planner.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from __future__ import division
+
+from datetime import datetime, timedelta
+from logging import getLogger
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.safe_eval import safe_eval
+
+_logger = getLogger(__name__)
+
+
+class CrmPhonecallPlan(models.TransientModel):
+    _name = "crm.phonecall.planner"
```

Comment:
The wizard needs an access rules definition. Otherwise, the wizard menu won't be accesible to anyone.